lib/utils.js

```diff
@@ -1,12 +1,13 @@
 const fs = require('fs')
 const {KeyValueStore} = require('./kv')
 module.exports.read = f => fs.readFileSync(f).toString('utf-8')
-module.exports.extractKVBindings = (setFlagValues) => {
+module.exports.extractKVBindings = (kvSetFlags, kvFileFlags) => {
```

Comment:
Sorry I missed this on the first CR but please add tests for the new argument and logic.

Reply:
Added testing now